### PR TITLE
[14.0][MIG] ddmrp

### PR DESCRIPTION
**Pull request by DavidBForgeFlow**
Standard migration to v14. Mantaining dependency on stock_helper

### CODE REVIEW

**Comment by LoisRForgeFlow** [comment]
* [x] : Depends on https://github.com/OCA/web/pull/1809.

File: ddmrp/views/stock_buffer_profile_view.xml [hunk 1]
```diff
@@ -8,6 +8,9 @@
                 <sheet>
                     <div class="oe_edit_only">
                         <label for="name" class="oe_inline" />
+                        <h1>
+                        <field name="name" />
+                        </h1>
```
Comment (LoisRForgeFlow) [inline ddmrp/views/stock_buffer_profile_view.xml]: Why is `name` added twice?

Comment (LoisRForgeFlow) [reply ddmrp/views/stock_buffer_profile_view.xml]: Nevermind, the first one is a `label` :laughing: nice catch!

**Comment by paulius-sladkevicius** [comment]
Static check of migrated code (f331ffb14d11aefd96d9f336349e8cf296a21922) looks good! As I'm not familiar with functionality (yet) I can't do functional testing.

**Review by JordiBForgeFlow — APPROVED** [review]

**Review by LoisRForgeFlow — APPROVED** [review]
Let's move forward!

**Pull request merged**